test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -20,8 +20,11 @@
   </ItemGroup>
     <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
         <PackageReference Include="NUnit" Version="2.6.3" ExcludeAssets="none" GeneratePathProperty="true" />
-        <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
-       
+        <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" /> 
+        <PackageReference Include="DynamoVisualProgramming.Analytics" Version="3.0.1.740">
```

Comment:
There's just one analytics package for both net48 and net6.0, which is the latest - 4.0.0.1432.